src/lib/sw_engine/tvgSwRle.cpp

```diff
@@ -612,7 +612,6 @@ static int _genRle(RleWorker& rw)
     return -1;              //lack of cell memory
 }
 
-
```

Comment:
Please don't change unnecessarily.
We keep 2 lines.

Reply:
ok, thank you